Systems/engine.xml

```diff
@@ -609,6 +609,52 @@
         <output>
             <property>/fdm/jsbsim/propulsion/engine[1]/volumetric-efficiency</property>
         </output>
-    </filter>            
+    </filter>
+
+    <!-- ============================================================== -->
+    <!-- Complex engine procedures                                      -->
+    <!-- ============================================================== -->
+
+    <logic>
+        <name>Complex engine procedures => oil consumption allowed</name>
```

Comment:
No problems. I will work on the About dialog today, so when we are reviewing it we can sneak a little fix for this there.